### PR TITLE
Cis

### PR DESCRIPTION
**Pull request by ruvdrsti**
I've been writing some theory and some code in the CIS notebook. Could you evaluate the theory part to see if I grasp the concept?

### CODE REVIEW

**Comment by ruvdrsti** [comment]
I have an additional question as well, as a sanity check. The energy we are looking for is the energy in equation (7) of the notebook right?

File: checkpoint_CIS.tex [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+\begin{equation}\tag{3}
+\langle \Phi_i^a |\hat{H}| \Phi_j^b \rangle = E_0\delta_{ij}\delta{ab} + F_{ab}\delta_{ij} - F_{ij}\delta_{ab} + \langle aj|| ib \rangle
+\end{equation}
+Now that we have an expression for our hamiltonian, we can solve the eigenequations for it. The only thing left to figure out is what basis this hamiltonian is given in. Since it is made up out of the ground state and the excited states, these should all be eigenfunctions of the hamiltonian itself, wo the basis used is MO basis. We could start using the eigenfunctions of the fock operator at scf conditions (which is off course needed for Brillouin's theorem to hold) and start iterating from there. So we would calculate the eigenfunctions and the energy, use these to create a new operator and repeat. Alternatively we could substract $E_0$ from it, so that we get the change in energy with respect to the orginal HF energy. An additional note we can make is that $F_{ij} = \delta_{ij}\epsilon_i$ at scf conditions in MO basis, since the fock matrix should be diagonal there. This will simplify Equation (3) to Equation (4), in which we also substracted $E_0$.
```
Comment (johdvos) [inline checkpoint_CIS.tex:21]: Be careful not to mix up the matrix elements of the **Hamiltonian matrix** (which are expectation values) and the actual **Hamiltonian operator**, which is an operator and from which the expression was known from the beginning.

Comment (johdvos) [inline checkpoint_CIS.tex:21]: Related to the previous remark: have look into the expression of the Hamiltonian operator and the so called correlation component of the electronic Hamiltonian.

**Review by johdvos — COMMENTED** [review]
I left some comments you can look into, but I am confident you grasp the concepts of CIS

**Comment by johdvos** [comment]
> I have an additional question as well, as a sanity check. The energy we are looking for is the energy in equation (7) of the notebook right? (equation (11) in Sherrill)

It is indeed this energy expression we are interested in, but the energy difference opposed to the hartree-fock reference is also something interesting.

**Pull request merged**